Zend/tests/readonly_classes/readonly_enum.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Enums cannot be readonly
+--FILE--
+<?php
+
+readonly enum Foo
+{
+}
+
+?>
+--EXPECTF--
+Parse error: syntax error, unexpected token "enum", expecting "abstract" or "final" or "readonly" or "class" in %s on line %d
```

Comment:
Not pretty sure if I can add two bits.. Just wanted to note that it doesn't make pretty much sense to read error message `expected ... "readonly"` when the code is `readonly enum Foo`. Two `readonly`s are not allowed as stated in test above.

Reply:
Aha, ok, I didn't know that. Just walked around and this caught my eyes. 🙂 Thanks for response.